src/worker/core/GameSim.basketball/index.ts

```diff
@@ -5,7 +5,11 @@ import getInjuryRate from "./getInjuryRate";
 import type { GameAttributesLeague, PlayerInjury } from "../../../common/types";
 import GameSimBase from "../GameSimBase";
 import { maxBy, range } from "../../../common/utils";
-import PlayByPlayLogger from "./PlayByPlayLogger";
+import PlayByPlayLogger, {
+	type blockType,
+	type fgaType,
+	type fgMissType,
```

Comment:
Types should start with a capital letter, that's just the normal convention everyone uses.